configure.ac

```diff
@@ -11,8 +11,8 @@ AC_PREREQ([2.63])
 define([scm_r], esyscmd([sh -c "git rev-parse --short=7 HEAD"]))
```

Comment:
OK, got you. I can add this check, no problem.
But what value should I use for `SCM_VERSION` in this case? Empty? `0000000`?